docs/getting-started/supported-languages-and-tools.md

```diff
@@ -193,7 +193,7 @@ The table below lists all languages and frameworks that Codacy supports and the
     <tr>
       <td>Markdown</td>
       <td><a href="https://github.com/remarkjs/remark-lint">RemarkLint</a></td>
-      <td></td>
+      <td><a href="https://github.com/markdownlint/markdownlint">MarkdownLint</a></td>
```

Comment:
Add the missing "wrench icon" with a link to an explanation of the suggested fixes feature:

![image](https://user-images.githubusercontent.com/60105800/119945355-3b3bc980-bf8d-11eb-9411-b6315e6c69e9.png)

```suggestion
      <td><a href="https://github.com/markdownlint/markdownlint">markdownlint</a>   <a href="#suggested-fixes">🔧</a></td>
```